azure_monitor/src/azure_monitor/export/metrics/__init__.py

```diff
@@ -99,3 +112,43 @@ def _metric_to_envelope(
         data = protocol.MetricData(metrics=[data_point], properties=properties)
         envelope.data = protocol.Data(base_data=data, base_type="MetricData")
         return envelope
+
+
+def standard_metrics_processor(envelope):
+    data = envelope.data.base_data
+    if data.metrics:
+        properties = {}
+        point = data.metrics[0]
+        if point.name == "http.client.duration":
+            point.name = "Dependency duration"
+            point.kind = protocol.DataPointType.AGGREGATION.value
+            properties["_MS.MetricId"] = "dependencies/duration"
```

Comment:
Looks like we are getting more AI properties like this one and others, it would be good to have them somewhere as constants to avoid bugs regarding typos

Reply:
Moving AI properties to a constants file I feel implies that they are a "global" accessible variable. These properties are only relevant to standard metric implementation and so should exist only in that scope (within the metrics file). It will be more suitable to define a constant at the top of this module.